app/assets/stylesheets/pageflow/themes/default/page.scss

```diff
@@ -253,3 +263,47 @@ $page-content-text-line-height: 1.5em !default;
     }
   }
 }
+
+
+h1,
+h2 span.subtitle,
+h2 span.tagline,
+h3,
+p {
+  max-width: $page-subtitle-tagline-max-width;
+  width: 60%;
+
+  @include mobile {
+    width: 100%;
+  }
+}
+
+// Styles when the text content is aligned right
+
+.text_position_right {
+  &.page .contentText > * {
+    margin-left: auto;
+  }
+
+  &.page {
```

Comment:
He wants you to merge the rules such that the `&.page` bit is not repeated from line 284:
```
&.page {
  .contentText > * {
    ...
  }

  h1,
  ... {
  }
}
```

instead of:

```
&.page .contentText > * {
   ...
}

&.page {
  h1,
  ... {
  }
}
```